app/templates/errors/410.html

```diff
@@ -0,0 +1,23 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Service no longer available - Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+<div class="error-page">
+    <div class="index-page grid-row">
+        <div class="column-two-thirds">
+            <header class="page-heading-smaller">
+                <h1>Service no longer available</h1>
+            </header>
+            <p>
+                The requested service was on a framework that is no longer live and so it is no longer available through the Digital Marketplace.
```

Comment:
I did think about that.  The first message I wrote was a generic "this isn't here any more", but then decided a service-specific message might be nicer as that's the only thing we have that makes 410s.  But yeah in that case it probably shouldn't be a generic error page.